src/containers/about/AboutSection.js

```diff
@@ -1,31 +1,69 @@
 import React from 'react';
 import styled from 'styled-components';
 
+import AboutImage from '../../assets/images/about/headshot.jpg';
 import PageSection from '../../components/layout/PageSection';
-
 import { NEUTRALS } from '../../core/styles/Colors';
 
 /* styled components */
 const Content = styled.div`
   align-items: center;
   display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  color: ${NEUTRALS.DARK_TEXT};
-  font-size: 86px;
-  font-weight: 200;
+  flex-direction: column;
   justify-content: center;
-  margin: 40px 140px;
-  max-width: 100%;
-  padding: 100px 0px;
   text-align: center;
 `;
 
+const TextContainer = styled.div`
+  font-size: 20px;
+  font-weight: 200;
+  margin: 0 50px;
+  max-width: 500px;
+`;
+
+const faceImageStyles = {
+  borderRadius: '50%',
+  margin: '60px',
+  maxWidth: '80%',
+  width: '414px',
+};
```

Comment:
This is an interesting way of doing it—you could also create a styled component for the `img` tag, and then you wouldn't have to pass along a styles object:
```
const FaceImage = styled.img
    borderRadius: 50%;
    margin: 60px;
    maxWidth: 80%;
    width: 414px;
;

...
<FaceImage src={AboutImage} />

Reply:
don't forget the backticks for the `styled.img` though—i couldn't put them in because github code formatting is weird.